system-int-test/test/cmr/system_int_test/search/collection_sorting_search_test.clj

```diff
@@ -504,4 +504,16 @@
            ["usage_score" "entry_title"] [c6 c7 c1 c2 c3 c5 c4]
            ["usage_score" "-entry_title"] [c7 c6 c1 c2 c3 c5 c4]
            ["-usage_score" "entry_title"] [c4 c5 c3 c2 c1 c6 c7]
-           ["-usage_score" "-entry_title"] [c4 c5 c3 c2 c1 c7 c6]))))
+           ["-usage_score" "-entry_title"] [c4 c5 c3 c2 c1 c7 c6]))
+
+    (testing "sorting params"
+      (are [query-map items]
+           (d/refs-match-order? items
+                                (search/find-refs :collection
+                                                  query-map
+                                                  {:method :post}))
+
+           {:keyword "a*"} [c1 c6]
+           {:keyword "a*" :sort-key ["-usage_score"]} [c1 c6]
+           {:keyword "a*" :sort-key ["usage_score"]} [c6 c1]))))
```

Comment:
We should add a couple tests to show that `usage_score` sort key takes precedence over other sort_keys on keyword searches when multiple sort keys are present. e.g.
```
{:keyword "a*" :sort-key ["entry_title" "usage_score"]} [c6 c1]
{:keyword "a*" :sort-key ["-entry_title" "usage_score"]} [c6 c1]
{:keyword "a*" :sort-key ["entry_title" "-usage_score"]} [c1 c6]
{:keyword "a*" :sort-key ["-entry_title" "-usage_score"]} [c1 c6]
```